src/java.base/share/classes/java/lang/constant/DynamicCallSiteDesc.java

```diff
@@ -41,12 +41,11 @@
  * A <a href="package-summary.html#nominal">nominal descriptor</a> for an
  * {@code invokedynamic} call site.
  *
- * <p>Concrete subtypes of {@linkplain DynamicCallSiteDesc} should be immutable
- * and their behavior should not rely on object identity.
+ * <p>This class is immutable and its behavior does not rely on object identity
```

Comment:
```suggestion
 * <p>A {@code DynamicCallSiteDesc} is immutable and its behavior does not
 * rely on object identity.
```
This describes an object, not a class, and please close sentences with a period.
(We use fragments for param and return block tags instead)

Reply:
Thanks, fixed in 6156b46 :)